data.sql

```diff
@@ -41,4 +41,40 @@ UPDATE animals SET owners_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owners_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
 UPDATE animals SET owners_id = 3 WHERE name IN ('Devimon', 'Plantmon');
 UPDATE animals SET owners_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom');
-UPDATE animals SET owners_id = 5 WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+UPDATE animals SET owners_id = 5 WHERE name IN ('Angemon', 'Boarmon');
+
+
+-- day 4
+INSERT INTO vets (name, age, date_of_graduaction) VALUES 
+('William Tatcher', 45, '2000-04-23'),
+('Maisy Smith', 26, '2019-01-17'),
+('Stephanie Mendez', 64, '1981-05-04'), 
+('Jack Harkness', 38, '2008-06-08');    
+
+INSERT INTO specializations (species_id, vets_id) VALUES 
+(1, 1), 
```

Comment:
[OPTIONAL]
- [ ] Instead of hard coding the values into specializations table I would suggest the dynamic way because What if it was some big data and don't know the `ids` ?
Hint: This can be achieved by saying:
```sql
INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));
```